assets/sass/06-components/navigation.scss

```diff
@@ -89,34 +88,41 @@
 	margin-top: 0;
 	margin-bottom: 0;
 
-	@include media(mobile-only) {
-		width: 100%;
-	}
-
 	// Mobile menu closed
 	> .primary-menu-container {
+		position: absolute;
 		visibility: hidden;
 		opacity: 0;
-		position: fixed;
 		top: 0;
 		right: 0;
 		bottom: 0;
 		left: 0;
-		padding: calc(4 * var(--global--spacing-unit)) var(--global--spacing-unit) var(--global--spacing-horizontal);
+		padding: calc(3.5 * var(--global--spacing-unit)) var(--global--spacing-unit) var(--global--spacing-horizontal);
 		background-color: var(--global--color-background);
-		overflow-x: hidden;
+		overflow-x: scroll;
 		overflow-y: auto;
 		transition: all .15s ease-in-out;
 		transform: translateY(var(--global--spacing-vertical));
 
 		@include media(mobile-only) {
+			height: 100vh;
 			z-index: 499;
 		}
+
+		.admin-bar & {
+			height: calc(100vh - 46px);
```

Comment:
This is height is affecting the wide menu as well.

Reply:
ah, thanks for catching that. Fixed in https://github.com/WordPress/twentytwentyone/pull/286/commits/32c420deb097acd8ea1b26db43c7c6b3ae5336ff